dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -90,6 +90,7 @@
             <Color x:Key="SolidBackgroundFillColorSecondary">#1C1C1C</Color>
             <Color x:Key="SolidBackgroundFillColorTertiary">#282828</Color>
             <Color x:Key="SolidBackgroundFillColorQuarternary">#2C2C2C</Color>
+            <Color x:Key="SolidBackgroundFillColorTransparent">#00202020</Color>
```

Comment:
If you say that Color animations animate using RGB channels but not the Alpha channel - then how does adding transparent version of the base color (using **alpha** value) solves the  issue?:)

Reply:
We now animate from this value to the base color instead of from 00FFFFFF which results in the white flash.